src/app/not-found.tsx

```diff
@@ -1,5 +1,28 @@
+import Image from "next/image";
+import image1 from "../../public/404.webp";
+import Link from "next/link";
+
 const NotFound = () => {
-  return <div>404 Page Not Found</div>;
+  return (
+    <div className="relative h-screen w-full overflow-hidden bg-hearts-beige">
+      <Image
+        src={image1}
+        alt="404 Background"
+        fill
```

Comment:
Make sure to not use pure css like fill and style={...}. Only use tailwindcss. Tailwindcss have "w-full" and "object-contain" which do the same as those.

Reply:
Can also add a mt-[5%] on the parent div tag to put the image at the better spot. Not sure if you need it but just an idea :)